README.md

```diff
@@ -1,24 +1,58 @@
-# README
+# Blog app
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> **Blog_app** is an implementation of the classic blog website. It's a full stack ruby on rails website that show the list of posts and empower readers to interact with them by adding comments and liking posts.
 
-Things you may want to cover:
+## Technologies
 
-* Ruby version
+- Ruby on Rails
+- Rspec
+- Rubocop
 
-* System dependencies
+## Milestones
 
-* Configuration
+- [x] Milestone 1: Setup and controllers.
+- [x] Milestone 2: Controllers specs.
+- [x] Milestone 3: Creating a data model.
+- [x] Milestone 4: Processing data in models.
+- [x] Milestone 5: Views.
+- [x] Milestone 6: Forms
+- [x] Milestone 7: Validations, Model specs, and n+1 problems.
+- [x] Milestone 8: Add Devise.
+- [x] Milestone 9: Add authorization rules.
+- [x] Milestone 10: Integration specs for Views.
+- [ ] Milestone 11: Add API endpoints.
+- [ ] Milestone 12: API documentation.
 
-* Database creation
+## ERD diagram
 
-* Database initialization
+![ERD](./doc/blog_app_erd.png)
 
-* How to run the test suite
+## Getting Started
 
-* Services (job queues, cache servers, search engines, etc.)
+- Run `bundle install` to get the dependencies.
+- Run `rails s` to start interactive the `puma` server.
+- Open `http://localhost:3000` in your browser.
 
-* Deployment instructions
+- to run tests `bundle exec rspec`.
 
```

Comment:
- [ ] [Optional] It would be nice to add instructions for setting up the database and other necessary actions before getting the application running. For example:

```
check the ruby version being used and install it if it is not the same as that of the local machine
set up the database and direct them to the config/database.yml file
run 'bundle install'
run 'npm install' or 'yarn install'
run 'rails db:create'
run rails 'db:migrate'
run rails 'db:seed' if there is data in the seed
run 'rails s' to start the server locally
```